app/clients/letter/dvla.py

```diff
@@ -80,5 +82,59 @@ def authenticate(self):
 
         return response.json()["id-token"]
 
+    def reset_api_key(self):
```

Comment:
How do you feel about `change` instead of `reset`?

Reply:
yeah i think that's reasonable. do we think we should change that for the password function as well? No strong opinion but I'm leaning to yes - "reset password" often refers to where you don't know an old password. Change also matches the relevant sections in the DVLA api docs for both the password and api key methods

Reply:
yeah I think both too 👍

Reply:
squashed out 🦶